core/src/cluster_nodes.rs

```diff
@@ -480,11 +484,46 @@ pub fn make_test_cluster<R: Rng>(
     (nodes, stakes, cluster_info)
 }
 
+pub(crate) fn get_data_plane_fanout(shred_slot: Slot, root_bank: &Bank) -> usize {
+    if !enable_turbine_fanout_experiments(shred_slot, root_bank) {
+        return DATA_PLANE_FANOUT;
+    }
+    // Allocate ~2% of slots to fanout experiment.
+    match shred_slot % 359 {
+        11 => 64,
+        61 => 768,
+        111 => 128,
+        161 => 640,
+        211 => 256,
+        261 => 512,
+        311 => 384,
+        _ => DATA_PLANE_FANOUT,
+    }
```

Comment:
can we put this in an else block for safety's sake?

Reply:
done